docs/index.md

```diff
@@ -15,46 +15,50 @@ In order to integrate with Lead Safe, the health provider will need to make a RE
 ## API
 
 ### Submitting Prediction
```

Comment:
I would change this because a prediction is not submitted. (A patient record is being submitted; and the prediction is retrieved.)

Reply:
Agreed